.github/workflows/deploy-next.yml

```
@@ -0,0 +1,44 @@
+name: Deploy Next
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 */12 * * *'
+
+jobs:
+  deploy:
+    name: Deploy Next
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+
+      - name: Install Node
+        uses: actions/setup-node@v1
+        with:
+          node-version: 15
+
+      - name: Install pnpm
+        run: curl -L https://unpkg.com/@pnpm/self-installer | node
+
+      - name: Use node_modules cache
+        uses: actions/cache@v2
+        with:
+          path: '**/node_modules'
+          key: ${{ runner.os }}-modules-${{ hashFiles('**/pnpm-lock.yaml') }}
+
+      - name: Install deps
```

```suggestion
      - name: Install deps
         if: ${{ !steps.cache-restore.outputs.cache-hit }}
```

If the `node_modules` are already cached and are restored, then why should we run this step and waste that [precious precious 1 ms](https://github.com/cordis-lib/cordis/runs/3408711784?check_suite_focus=true)

---

kek